kotlinx-coroutines-core/native/src/WorkerMain.native.kt

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines
+
+internal expect inline fun workerMain(block: () -> Unit)
```

Comment:
Why does this function belong to the `main` source-set instead of `test` one?
It's used only from within tests

Reply:
It is used from the main code in `native-mt` branch and I'm trying to minimize the diff in `native-mt` branch.

Reply:
Added comment.